docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md

```diff
@@ -9,24 +9,74 @@ modeled in [Decision Model and Notation](https://www.omg.org/dmn/) (DMN).
+use your own implementation for a business rule task, see the **Alternative task implementation**
+section below. The sections before it apply to the DMN decision implementation only.
+:::
+
+When the process execution arrives at a business rule task, a decision is evaluated using the
+internal DMN decision engine. Once the decision is made, the process instance continues.
+
+If the decision evaluation is unsuccessful, an [incident](/components/concepts/incidents.md) is
+raised at the business rule task. When the incident is resolved, the decision is evaluated again.
+
+## Defining a called decision
+
+A called decision links the business rule task to a DMN decision. It can be defined using the
+`zeebe:calledDecision` extension element.
+
+A business rule task must define the DMN decision id of the called decision as `decisionId`.
+Usually, the `decisionId` is defined as a static value (e.g. `shipping_box_size`), but it can also
+be defined as an [expression](/components/concepts/expressions.md) (e.g. `= "shipping_box_size_" +
+countryCode`). The expression is evaluated on activating the business rule task (or when an incident
+at the business rule task is resolved), after input mappings have been applied. The expression must
+result in a `string`.
 
-The differences between these task types are the visual representation (i.e. the task marker) and
-the semantics for the model.
+A business rule task must define the process variable name of the decision result as
+`resultVariable`. The result of the decision will be stored in this variable. The `resultVariable`
+is defined as a static value.
 
-When a process instance enters a business rule task, it creates a corresponding job and waits for
-its completion. A job worker should request jobs of this job type and process them. When the job is
-completed, the process instance continues.
+## Variable mappings
+
+By default, the variable defined by `resultVariable` is merged into the process instance. This
+behavior can be customized by defining an output mapping at the business rule task.
+
+All variables in scope of the business rule task are available to the decision engine when the
+decision is evaluated. Input mappings can be used to transform the variables into a format accepted
+by the decision.
 
 :::note
-Jobs for business rule tasks are not processed by Zeebe itself. To process them, you must provide a job worker.
+Input mappings are applied on activating the business rule task (or when an incident at the business
+rule task is resolved), before the decision evaluation. When an incident is resolved at the business
+rule task, then the input mappings are applied again before evaluating the decision. This can affect
+the result of the decision.
 :::
 
-## Defining a task
+For more information about this topic visit the documentation about [Input/output variable
+mappings](/components/concepts/variables.md#inputoutput-variable-mappings).
+
+## Alternative task implementation
```

Comment:
:wrench: We could rename the section to `Alternative job worker implementation`. This name would align with the option in the modeler.

Reply:
🤔 I feel like it describes the implementation of the task, and there are currently 2 options: `DMN decision`, `Job worker`. It makes me wonder whether we should call it `Alternative` in the first place.

❓ What do think about `Job worker implementation` as the section header?

Reply:
Sounds good :+1: